STACpopulator/stac_utils.py

```diff
@@ -11,30 +10,6 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def url_validate(target: str) -> bool:
```

Comment:
We don't need to pre-validate URLs, the request library will do that for us and raise a much more informative error.